docs/index.Rmd

```diff
@@ -236,4 +238,48 @@ When `tar_make` runs without error on your machine, everything in your pipeline
 
 Typically if `tar_make` runs without error on one machine but not another, it is due to variation in package versions. See [guideline-reproducibility](https://github.com/WFUSM-BDS/guideline-reproducibility) for information on the `renv` package and how package management is essential to creating a reproducible pipeline on all machines.
 
-<BR>
\ No newline at end of file
+<BR>
+
+# Coding practices with targets
+
+## File organization
+
+Each project will look slightly different depending on its purpose, many projects may include these files/folders
+
+- **R/** - includes all R programs
+
+- **reports/** - includes .Rmd or .qmd files that render reports such as web reports or DSMB reports
+
+- **\_targets.R** - this R file contains your targets pipeline
+
+- **.gitignore** - Be sure to add _targets/ to the .gitignore so that you don't push any data
+to GitHub accidentally. See the [guideline-data-safety](https://github.com/WFUSM-BDS/guideline-data-safety?tab=readme-ov-file#using-gitignore) repo for more information about .gitignore
+
+- **packages.R** - An R program to load all packages into R session. This should be sourced in _targets.R
+
+For a larger example of what a reporting pipeline for a Data Coordinating Center could look like resulting in a DSMB report, see [leap-report](https://github.com/WFUSM-BDS/leap-report?tab=readme-ov-file#leap-report)
+
+## Naming targets
+
+- Your project should set some guidelines for naming based off of your anticipated pipeline. Below are suggestions for what the first word of a target should be named and what it should generally do.
+
+    + `data`: these targets create a dataset to be used downstream.
+    
+    + `result`: these targets leverage a dataset to make some result
+    
+    + `tbl`: these targets leverage a result to make a table object (e.g., a `flextable`)
+    
+    + `fig`: these targets leverage a result to make a figure (e.g., a `ggplot2` object)
+
+    + `report` : these targets create a report document
+
```

Comment:
Should we add some thoughts about suffixes for ending target names? We haven't really discussed this yet, so it may be something we need to talk about before we come up with a good policy. The suffixes are also a little more tricky because there is usually a set of suffixes that apply based on the prefix. 

For example, targets that start with `data` may have suffixes like this:

- `init`: this means the data target is initialized here, and will be modified in a later step
- `excluded`: this indicates exclusions have been applied for this data target
- `derived`: this means new variables have been added to the data (e.g., a calculated variable like eGFR)
- `analysis`: this means the data target can be used in analyses

Reply:
Feel free to punt to a later issue if we want to formalize this.